GVRf/Framework/framework/src/main/jni/objects/textures/render_texture.cpp

```diff
@@ -23,215 +23,438 @@
 #include "eglextension/msaa/msaa.h"
 
 namespace gvr {
-RenderTexture::RenderTexture(int width, int height, GLTexture* tex) :
+GLFrameBuffer *RenderTexture::renderTexture_gl_read_buffer = nullptr;
```

Comment:
we are using eyeCaptureRenderTexture only in case of multiview, with oculus backend, in non-multiview case, we are using the texture coming from oculus swap chain in which we don't have render-texture instance.